authors/apps/comments/tests/base.py

```diff
@@ -0,0 +1,69 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from ..tests.test_data import (new_user, user_login,
```

Comment:
Thanks for the good work @wasibani-roy , However you have a lot of unused imports in this file, they include the User model, comment. You can always clean them up to improve readability.